src/components/views/rooms/RoomSettings.js

```diff
@@ -440,43 +495,65 @@ module.exports = React.createClass({
                                defaultChecked={this.state.areNotifsMuted}/>
                         Mute notifications for this room
                     </label>
-                    <label>
-                        <input type="checkbox" disabled={ !roomState.mayClientSendStateEvent("m.room.join_rules", cli) }
-                            onChange={this._onToggle.bind(this, "join_rule", "invite", "public")}
-                            defaultChecked={this.state.join_rule !== "public"}/>
-                        Make this room private
-                    </label>
-                    <label>
-                        <input type="checkbox" disabled={ !roomState.mayClientSendStateEvent("m.room.guest_access", cli) }
-                            onChange={this._onToggle.bind(this, "guest_access", "can_join", "forbidden")}
-                            defaultChecked={this.state.guest_access === "can_join"}/>
-                        Let guests join this room
-                    </label>
+                    <div className="mx_RoomSettings_settings">
+                        <h3>Who can access this room?</h3>
+                        <label>
+                            <input type="radio" name="roomVis" value="invite_only"
+                                disabled={ !roomState.mayClientSendStateEvent("m.room.join_rules", cli) }
```

Comment:
Technically this is a lie. If `guest_access` is `forbidden` then you need to flip it to be `can_join`, so you need to have permission to send `m.room.guest_access` too. (in which case, factor out the check please!)